File: app/src/main/java/com/redrock/modulizationlearn/MainActivity.java
package com.redrock.modulizationlearn;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.redrock.arouter_annotation.ARouter;
import com.redrock.arouter_api.RouterManager;


@ARouter(path = "/app/MainActivity/")
public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        RouterManager.getInstance()
                .build("/login/MainActivity/")
                .withString("str","Parameter")
                .navigation(this);
    }
}

File: app/src/main/java/com/redrock/modulizationlearn/MainActivity.kt
package com.redrock.modulizationlearn

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Toast.makeText(this,"服务器地址：${BuildConfig.debug}",Toast.LENGTH_LONG).show()


    }
}